deploy/charts/csi-driver/templates/daemonset.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       containers:
 
         - name: node-driver-registrar
-          image: quay.io/k8scsi/csi-node-driver-registrar:v2.1.0
+          image: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.1.0
```

Comment:
Let's bump to v2.3.0, don't think there is a reason not to.

Reply:
Okie doke, done now.